config/schema.error.yaml

```
@@ -0,0 +1,575 @@
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_SOLR_DEPRECATED:
+    title: 'Solr support has been deprecated in Magento 2.1.'
+    stage: deploy
+    step: validate-config
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_SOLR_NOT_SUPPORTED:
+    title: 'Solr is no longer supported by Magento 2.2 or later.'
+    stage: deploy
+    step: validate-config
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_ES_INSTALLED_BUT_NOT_USED:
+    title: 'Elasticsearch service is installed at infrastructure layer but is not used as a search engine.'
+    suggestion: 'Consider removing elasticsearch service from infrastructure layer for optimized resource usage.'
+    stage: deploy
+    step: validate-config
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_ES_VERSION_MISMATCH:
+    title: 'Elasticsearch service version on infrastructure layer is not compatible with current version of elasticsearch/elasticsearch module, used by your Magento application.'
+    stage: deploy
+    step: validate-config
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_CONFIG_NOT_COMPATIBLE:
+    title: 'The current configuration is not compatible with this version of Magento'
+    stage: deploy
+    step: validate-config
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_DEPLOY_SCD_OPTIONS_IGNORANCE:
+    title: 'Scd options ignored as deploy didn''t run on the build phase'
+    stage: deploy
+    step: validate-config
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_DEPRECATED_VARIABLES:
+    title: 'The configuration contains deprecated variables or values'
+    stage: deploy
+    step: validate-config
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_ENVIRONMENT_CONFIG_NOT_VALID:
+    title: 'Environment configuration is not valid'
+    stage: deploy
+    step: validate-config
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_CONFIG_WRONG_JSON_FORMAT:
+    title: 'Json type configuration can''t be decoded'
+    stage: deploy
+    step: validate-config
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_SERVICE_VERSION_NOT_COMPATIBLE:
+    title: 'The current configuration is not compatible with this version of Magento'
+    stage: deploy
+    step: validate-config
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_SERVICE_PASSED_EOL:
+    title: 'Some services have passed EOL'
+    stage: deploy
+    step: validate-config
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_DEPRECATED_MYSQL_SEARCH_ENGINE:
+    title: 'The MySQL search configuration option is deprecated'
+    suggestion: 'Use Elasticsearch instead'
+    stage: deploy
+    step: validate-config
+    type: warning
+!php/const Magento\MagentoCloud\App\Error::WARN_ENV_PHP_MISSED:
+    title: 'Magento state indicated as installed but configuration file app/etc/env.php was empty or did not exist.'
```

```suggestion
    title: 'Magento installation completed, but the `app/etc/env.php` configuration file was missing or empty.'
```

Not sure about context -- does the state indicated installed, but because of missing or empty env.php install was not completed?